src/Dev/TestSession.php

```diff
@@ -239,9 +239,9 @@ public function cssParser() {
 	 * @return SimplePage The response if available
 	 */
 	public function lastPage() {
-		require_once("thirdparty/simpletest/http.php");
```

Comment:
Everyone once and I while I realise that certain pieces of code are still in there :-P